CHANGELOG.md

```diff
@@ -1,7 +1,19 @@
-# Next Version
+# Next Version 13.0
```

Comment:
Would be nice if we can ...

* Use just master in the changelog version, we never know what's going to happen and the version can be misleading, we can make a branch from the latest release tag, fix a bug, introduce new thing and release as 0.13 and then the 0.13 in the master's changelog must be update to something else, like 0.14, etc.
* Split changes to logical group (by modules for example).
* Do not mention documentation changes.
* Do not use past tense, just `Remove` instead of `Removed`, etc.

Proposal ...

```text
# master

- `input` module
  - Derive 'Copy' for 'KeyEvent'
- `cursor` module
  - Remove `TerminalCursor`, `cursor`, `Crossterm::cursor()`
  - Remove root re-exports of the `cursor` module types, accessible via `crossterm::cursor` only
  - Introduce `crossterm::cursor::position` as a `TerminalCursor::pos` replacement
  - Rename `Goto` command to `MoveTo`
  - Rename `Up` command to `MoveLeft`
  - Rename `Right` command to `MoveRight`
  - Rename `Down` command to `MoveDown`
  - Rename `BlinkOn` command to `EnableBlinking`
  - Rename `BlinkOff` command to `DisableBlinking`
  - Rename `ResetPos` command to `ResetPosition`
  - Rename `SavePos` command to `SavePosition`
```

Reply:
Unresolved, check my proposal. Backticks in around Copy, ... rename vs renamed vs refactored vs ..., ... Also the doc note is probably not useful.